pkg/ccl/backupccl/backup_processor_planning.go

```diff
@@ -48,6 +48,8 @@ func distBackup(
 	dsp := execCtx.DistSQLPlanner()
 	evalCtx := execCtx.ExtendedEvalContext()
 
+	// We don't return the compatible nodes here since PartitionSpans will
```

Comment:
not a change for this PR but now I'm sorta wondering why we even SetupAllNodesPlanning if we're not planning on _all_ nodes

Reply:
we still need SetupAllNodesPlanning here so that the planCtx can learn about the node status of all of the nodes. I think there's an argument to split out `getAllPlannableNodes` out again, which is just a pure function on the `planCtx`. We only need to `getAllPlannableNodes` in import and restore where we round-robin distribute work.

Even in backup, we still need `planCtx` to know about the status of all nodes so it only plans on healthy ones.

Reply:
Makes sense, thanks!